src/lib/idb/state.js

```diff
@@ -5,7 +5,7 @@ import Chats from './stores/Chats'
 import Security from './stores/Security'
 
 /** Modules that will be stored in IDB **/
-const modules = ['adm', 'eth', 'doge', 'bnb', 'bz', 'partners', 'delegates']
+const modules = ['adm', 'eth', 'doge', 'bnb', 'bz', 'contacts', 'delegates']
```

Comment:
Coin-modules (`eth`, `doge`, etc.) should be added dynamically (see `constants/Cryptos`).

Reply:
What do you suggest?
`const modules = ['partners', 'delegates'].concat(Object.values(Cryptos).map(crypto => crypto.toLowerCase()))`